core/amber/src/main/scala/edu/uci/ics/texera/web/service/WorkflowJobService.scala

```diff
@@ -70,7 +70,8 @@ class WorkflowJobService(
     if (WorkflowService.userSystemEnabled) {
       workflowContext.executionID = ExecutionsMetadataPersistService.insertNewExecution(
         workflowContext.wId,
-        workflowContext.userId
+        workflowContext.userId,
+        workflowContext.engineVersion
```

Comment:
you don't need to add engineversion in workflow context, it is a one time thing, for the execution request. can be passed only and not saved in the workflowcontext

Reply:
removed